Source/Scene/PointCloudPostProcessor.js

```diff
@@ -222,15 +315,59 @@ define([
 
         if (resized) {
             destroyFramebuffers(processor);
-            createFramebuffers(processor, context);
-            updateCommandFramebuffers(processor);
+            //createFramebuffers(processor, context);
```

Comment:
We generally don't commit code that is commented out.  Usually, remove it.

Otherwise, it can stay in your branch like this, but we would remove it before merging into `master`.